### DISCUSSION TITLE
Changelog for beta versions

### DISCUSSION THREAD

**Discussion by desiboli** [amannn/next-intl#497] [General]
Anywhere I can see the changelog for the beta version releases ? 🙏

**Comment by amannn** [comment]
Please see https://github.com/amannn/next-intl/issues/485#issuecomment-1697463116

I didn't have time to set up a proper release channel for the betas so far, but also didn't expect the beta to run for so long to be honest 🙂. I hope that we can move to stable releases in the foreseeable future, so this is more transparent.
